src/styl/_base/_typography.styl

```diff
@@ -32,16 +32,19 @@ h5
 h6
     _fontload($fontstack-headings)
     font-weight bold
-    line-height 1.2
+    line-height _lh(1)
```

Comment:
As the formula of `_lh` is $lines * $line-height
Where $line-height is $lh (`_rem(24px)`) how is this line relevant ?